.github/workflows/cargo.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Cargo
+
+on:
+  pull_request:
+    branches:
+    - master
+    paths:
+      - 'bazel/cargo/**'
+
+  push:
+    branches:
+    - master
+    paths:
+      - 'bazel/cargo/**'
+
+jobs:
+
+  format:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Format (cargo raze)
+      working-directory: bazel/cargo
+      run: |
+        cargo install cargo-raze --version 0.7.0
+        cargo raze
+        git diff --exit-code
```

Comment:
You need to add `cargo generate-lockfile` before `cargo raze` to catch minor updates and generate sha256 entries.

Reply:
done